src/axom/sina/tests/CMakeLists.txt

```diff
@@ -0,0 +1,119 @@
+
+foreach(test ${gtest_sina_tests})
+    get_filename_component( test_name ${test} NAME_WE )
+    axom_add_executable(NAME ${test_name}_test
+                        SOURCES ${test}
+                        OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
+                        DEPENDS_ON ${sina_gtests_depends_on}
+                        FOLDER axom/sina/tests
+                        )
+
+    axom_add_test( NAME    ${test_name}
+                   COMMAND ${test_name}_test
+                   )
+endforeach()
+
+
+if (ENABLE_GMOCK)
+
+    #------------------------------------------------------------------------------
+    # Create test utilities library for Sina
+    #------------------------------------------------------------------------------
+
+    # Define Sina test utility sources and headers
+    set(sina_test_utils_sources
+        SinaMatchers.cpp
+        TestRecord.cpp
+        )
+    set(sina_test_utils_headers
+        SinaMatchers.hpp
+        TestRecord.hpp
+        )
+
+    set(sina_test_utils_depends_on gmock ${sina_gtests_depends_on})
+
+    # Create a library for the test utilities so they can be used
+    axom_add_library(
+        NAME       sina_test_utils
+        SOURCES    ${sina_test_utils_sources}
+        HEADERS    ${sina_test_utils_headers}
+        DEPENDS_ON ${sina_test_utils_depends_on}
+        FOLDER     axom/sina/tests)
+
+    #------------------------------------------------------------------------------
+    # Add gmock C++ tests
+    #------------------------------------------------------------------------------
+
+    set(gmock_sina_tests
+        sina_ConduitUtil.cpp
+        sina_Curve.cpp
+        sina_CurveSet.cpp
+        sina_DataHolder.cpp
+        sina_Datum.cpp
+        sina_Document.cpp
+        sina_ID.cpp
+        sina_Record.cpp
+        sina_Relationship.cpp
+        sina_Run.cpp
+        )
+
+    set(sina_gmock_depends_on sina_test_utils)
+
+    # Add tests using Adiak if necessary and Adiak dependency
+    blt_list_append( TO gmock_sina_tests ELEMENTS sina_AdiakWriter.cpp IF AXOM_USE_ADIAK)
+    blt_list_append( TO sina_gmock_depends_on ELEMENTS adiak::adiak IF AXOM_USE_ADIAK )
+
+    foreach(test ${gmock_sina_tests})
+        get_filename_component( test_name ${test} NAME_WE )
+        axom_add_executable(NAME ${test_name}_test
+                            SOURCES ${test}
+                            OUTPUT_DIR ${TEST_OUTPUT_DIRECTORY}
+                            DEPENDS_ON ${sina_gmock_depends_on}
+                            FOLDER axom/sina/tests
+                            )
+
+        axom_add_test( NAME    ${test_name}
+                       COMMAND ${test_name}_test
+                       )
+    endforeach()
+endif()
+
+#------------------------------------------------------------------------------
+# Add fortran integration test
+#------------------------------------------------------------------------------
+if (ENABLE_FORTRAN)
+    find_package(Python REQUIRED COMPONENTS Interpreter)
```

Comment:
Ok, I haven't pushed the change just yet but I've modified this to remove the `find_package` and `if (Python_found)` statement.

Reply:
@kennyweiss @white238 after making changes to this it looks like it's causing the test suite to fail since it can't find a python executable. The current code now looks like so:
![image](https://github.com/user-attachments/assets/7ab20865-e243-44c6-b5c6-54525226df2b)

Is there a way to point the test suite to a python executable? Should we go back to how this was previously?

Reply:
I think the problem is that several of our host-configs don't provide a `PYTHON_EXECUTABLE`, 
e.g. the docker images we use for our Azure tests, https://github.com/LLNL/axom/blob/develop/host-configs/docker/gcc%4013.1.0.cmake, and the CI on Tioga (https://lc.llnl.gov/gitlab/axom/axom/-/pipelines/474746)

So, a quick fix would be to add a guard to only run the Sina Fortran test when a `PYTHON_EXECUTABLE` is provided.

It looks like we should also add the `+python` variant to Axom's spack config for our docker images -- https://github.com/LLNL/axom/blob/9c1a5ce3b97009f70ff6b720f568974b1e7f9c44/scripts/spack/packages/axom/package.py#L557-L561
(@white238 -- is there a reason that we don't currently do this?)